howtos/oem-guide.rst

```diff
@@ -0,0 +1,135 @@
+
+For our software license, please see FAQ entry: ":ref:`faq_license`".
+Compliance with the stated licenses under which our code is released,
+and their necessarily associated Copyright notices,
+and that of our trivially modified linux distro base; the "Built on openSUSE" bit,
+is a legal requirement.
+Note however; this requirement does not differ from any other predominantly GPL-3.0-or-later licensed software bundling.
+I.e. one is free to redistribute, as-is, so long as no modifications are made.
+Modified versions, once re-distributed,
+entail further legal responsibilities regard availability of the modified version.
+Continuing the freedoms originally granted, for folks to know what code their systems are running.
+
+Note that `The GPL Is a License, not a Contract <https://lwn.net/Articles/61292/>`_.
+
+`The Rockstor Project <https://opencollective.com/the-rockstor-project>`_
+is an `Open Collective <https://opencollective.com/>`_
+Non-Profit/Non-Business Open Source community endeavour.
+
+- All rockstor distributed code is FSF Free/Libre & OSI approved and available on our
+  `GitHub Organisation <https://github.com/rockstor>`_ site.
+
+- All 'rockstor' rpm packages are source (--format sdist) in-nature: i.e. non binary.
+
+- All released packages are build directly from publicly tagged GitHub releases,
+  for details see: `rockstor-rpmbuild <https://github.com/rockstor/rockstor-rpmbuild>`_.
+
+- All installers are built using an open source project and configuration,
+  for details see: `rockstor-installer <https://github.com/rockstor/rockstor-installer>`_.
+
+Our continued existence, i.e. ongoing-development & distribution services,
+depend entirely upon good-will and active community participation in our Open Collective membership Tiers.
+As such circumvention of these service orientated tiers (curated release channels / support / membership / etc),
+in any way, would result in no future communication or service to the offending agent.
+We protect, by way of our licensing choices, and consequent code availability, the end-users freedoms.
+That does not oblige any member of The Rockstor Project's team in any way: regardless of Tier membership.
+
+**Given the above freedoms, all-around, we can only request the following:**
+
+1. OEM hardware distributors/redistributors resellers honour our existing sustainability model.
+I.e. they have an appropriate number, given anticipated development team contacts per-year,
+of 'OEM Pre-installer subscription' memberships.
+Rockstor's continued development, and existence,
+depends entirely on interested parties appreciating both past and ongoing development team efforts and associated costs.
+
+The Rockstor Project development team, in-turn,
+will endeavour to prioritise OEM/reseller technical enquiries/requests as resources allow.
+I.e. a pre-installer's enquiries/concerns are assumed to relate to more than one install,
+where-as our non OEM contribution tiers relate to specific installs; i.e:
```

Comment:
whereas our non-OEM contribution tiers relate to specific installs, which are as follows: